### DISCUSSION TITLE
Rainfall infiltration example

### DISCUSSION THREAD

**Discussion by SeyedSajjadMahdavi** [Anura3D/Anura3D_OpenSource#74] [Q&A] [answered]
**Hello Dear all,**
I am learning how to model slope failure in Anura3D caused by _Rainfall infiltration_ as explained in the Tutorial. Everything looks great but at the end when I input vtk files in the ParaView and go through playing animation, **slope material points don't move**. I double check all the processes also as described in Youtube, but nothing changed. I should mention that the _oedometer_ example was conducted with no problem.
I'd really appreciate, if providing me the probable reasons?
![1](https://user-images.githubusercontent.com/106150321/222476756-5199ddf0-2781-4e99-b515-5609beb1ba35.jpg)
![2](https://user-images.githubusercontent.com/106150321/222476789-04f84c72-1517-48e4-8911-31df883ae673.jpg)

**Bests,
Sajjad**

**Comment by FrancescaCeccato** [comment]
hello,
are displacements very small or exaclty 0?
Maybe this infiltration does not cause failure.

Did you activate the flag $$FIX_SOLID_SKELETON? it should be 0 if you want soil deformations and 1 if you are interested only in the seepage phenomenon (so soil deformations).

**Comment by SeyedSajjadMahdavi** [comment]
**Hi Dr Ceccato,**
after a long time! Thanks for your great response
I checked the flag you mentioned it is 0, also the displacements are not exactly zero (fig bellow). So I'm still wondering why this happen. 

I made one model with the real cohesion of soil and it's permeability from the first stage (QUASISTATIC) then the slope failed although the displacements were not close to the desired in manual2022.
I also check one thing more. In CPS_002 file when I activate the flag $$APPLY_MATERIAL_UPDATE in the following CPS files it would be again inactivated! I don't know whether it might be relevant or not.
I really need your help. 
BESTS
![Screenshot (36)](https://github.com/Anura3D/Anura3D_OpenSource/assets/106150321/c81d39ba-9728-4490-9ff5-a2c3af8e41ab)

**Comment by FrancescaCeccato** [comment]
The displacement of your picture are typical of gravity loading. During gravity loading, cohesion should be high in order to ensure elastic behaviour of soil and the slope should not fail.
After gravity loading, you have to update the cohesion to its real value. This is done by updating the GOM and setting the flag $$APPLY_MATERIAL_UPDATE to 1. It will be 0 automatically at the following steps, this is correct.

**Reply by SeyedSajjadMahdavi** [reply]
Right, I do change the required settings in the second phase. Since I think the only input files may cause this issue, attached them and would appreciate if  you take a look at them.
[CPS_002.pdf](https://github.com/Anura3D/Anura3D_OpenSource/files/11486664/CPS_002.pdf)
[GOM.pdf](https://github.com/Anura3D/Anura3D_OpenSource/files/11486667/GOM.pdf)

**Reply by FrancescaCeccato** [reply]
They look ok.

**Comment by SeyedSajjadMahdavi** [comment, marked as answer]
I'm happy to share that Rainfall infiltration model worked correctly on my system with 2023 release of Anura3D, thank you for all supports @FrancescaCeccato.
